src/main/java/Main.java

```diff
@@ -22,7 +22,7 @@ public static void main(String[] args) {
 
         System.out.println("Short report:");
         System.out.print(student1.name + " (" + student1.curriculum + ") - ");
-        AmountOfTimeBeforeOrAfterCourseCompletion.calculateAmountOfTimeBeforeOrAfterCourseCompletion((DetermineCourseCompletion.getCourseCompletionDate(startDateStudent1, (course1_1.duration + course1_2.duration + course1_3.duration))), currentDate, (course1_1.duration + course1_2.duration + course1_3.duration));
+        AmountOfTimeBeforeOrAfterCourseCompletion.calculateAmountOfTimeBeforeOrAfterCourseCompletion((DetermineCourseCompletion.getCourseCompletionDate(startDateStudent1, (course1_1.duration + course1_2.duration + course1_3.duration))), currentDate);
```

Comment:
Can you implement `Curriculum` class in order to avoid sum expressions: `(course1_1.duration + course1_2.duration + course1_3.duration)`?